index.js

```diff
@@ -36,6 +36,9 @@ function formatName (arr) {
 }
 
 function processString (str, system) {
+  if (!str) {
+    throw new Error('The string is null');
```

Comment:
:+1: but a more descriptive message would be better. What about "Input contains non-Chinese character"?